docs/Release_Management.md

```diff
@@ -0,0 +1,68 @@
+# Release Management
+
+## Overview
+
+This document describes KAITO project release management, which includes release versioning, supported releases, and supported upgrades.
+
+## Legend
+
+- **X.Y.Z** refers to the version (git tag) of KAITO that is released. This is the version of the KAITO images and the Charts version.
+- **Breaking changes** refer to schema changes, flag changes, and behavior changes of KAITO that may require a clean installation during upgrade, and it may introduce changes that could break backward compatibility.
+- **Milestone** should be designed to include feature sets to accommodate 2 months release cycles including test gates. GitHub's milestones are used by maintainers to manage each release. PRs and Issues for each release should be created as part of a corresponding milestone.
+- **Patch releases** refer to applicable fixes, including security fixes, may be backported to support releases, depending on severity and feasibility.
+- **Test gates** should include soak tests and upgrade tests from the last minor version.
+
+## Release Versioning
+
+All releases will be of the form _vX.Y.Z_ where X is the major version, Y is the minor version and Z is the patch version. This project strictly follows semantic versioning.
+
+The rest of the doc will cover the release process for the following kinds of releases:
+
+**Major Releases**
+
+No plan to move to 1.0.0 unless there is a major design change like an incompatible API change in the project
```

Comment:
Should we move to 1.0.0 when we graduate the API from v1beta1 to v1?

Reply:
@Fei-Guo thoughts?